packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -1,23 +1,48 @@
 import React from 'react';
 import { Text, StaticUrl } from '@deriv/components';
-import { Localize } from '@deriv/translations';
+import { localize, Localize } from '@deriv/translations';
 import ListingContainer from 'Components/containers/listing-container';
 import './cfds-listing.scss';
 import { useStores } from 'Stores/index';
 import { observer } from 'mobx-react-lite';
 import AddOptionsAccount from 'Components/add-options-account';
 import { isMobile } from '@deriv/shared';
+import TradingAppCard from 'Components/containers/trading-app-card';
+import { AvailableAccount } from 'Types';
 
 const CFDsListing = () => {
     const { tradinghub } = useStores();
-    const is_real = tradinghub.selected_account_type === 'real';
-    const has_no_real_account = !tradinghub.has_any_real_account;
+    const {
+        available_dxtrade_accounts,
+        available_mt5_accounts,
+        selected_region,
+        selected_account_type,
+        has_any_real_account,
+        checkForExistingAccounts,
+        startTrade,
+        is_eu_user,
+    } = tradinghub;
+    const has_no_real_account = !has_any_real_account;
+    const is_real = selected_account_type === 'real';
+    const accounts_sub_text = is_eu_user ? localize('Account Information') : localize('Compare accounts');
     return (
         <ListingContainer
             title={
                 !isMobile() && (
                     <Text size='sm' line_height='m' weight='bold'>
-                        <Localize i18n_default_text='CFD Listing' />
+                        <Localize
+                            i18n_default_text='CFDs<0>{{accounts_sub_text}}</0>'
+                            values={{ accounts_sub_text }}
+                            components={[
+                                <Text
+                                    key={0}
+                                    color={'red'}
+                                    size='xxxs'
+                                    weight='bold'
+                                    styles={{ marginLeft: '1rem' }}
```

Comment:
can we remove this inline style

Reply:
@amina-deriv, this is the only style I'm adding to it, you want me to create a class for that?